pkgs/top-level/all-packages.nix

```diff
@@ -498,7 +507,7 @@ with pkgs;
   });
 
   androidenv = callPackage ../development/mobile/androidenv {
-    pkgs_i686 = pkgsi686Linux;
+    pkgs-i686 = pkgs-i686;
```

Comment:
Knowing that `callPackage` is used, this attribute can be removed, as it would be inherited from `self` anyway.